### DISCUSSION TITLE
ketcher-react documentation

### DISCUSSION THREAD

**Discussion by mustafacanp** [epam/ketcher#584] [Q&A] [answered]
What are the supported props of the `Editor` component on the react? I just found the hidden state of the buttons.

```
<Editor
    buttons={{
        layout: {hidden: true},
        clean: {hidden: true},
        arom: {hidden: true},
        dearom: {hidden: true},
        cip: {hidden: true},
        check: {hidden: true},
        analyse: {hidden: true},
        recognize: {hidden: true},
        miew: {hidden: true}
    }}
    staticResourcesUrl={staticResourcesUrl}
    structServiceProvider={structServiceProvider}
/>
```

Do you have documentation? If not, could you please give me some code examples about the supported properties?

**Comment by AndreiMazol** [comment, marked as answer]
Hi @mustafacanp ,

Ketcher is under redesign so we are focusing on code changes rather than documentation. Documentation will be created a little bit later.

Supported Editor properties:

'buttons' - it allows you to  hide any buttons (by default all buttons are visible)
'staticResourceUrl' - it defines the path where help.md and library.sdf files are located (they should be hosted as a part of your app)
'structServiceProvider' - it is responsible for creating of StructService instances. This moment two providers are available: RemoteStructServiceProvider (it requires Indigo Service - remote mode) and StandaloneStructServiceProvider (it is based on WASM and allows to execute backend functions on client side without any additional dependencies). Please look at 'example\src\App.tsx' for details.

Best regards,
Andrei

**Reply by mustafacanp** [reply]
Thanks @AndreiMazol
